style.css

```diff
@@ -78,8 +95,271 @@ h1 {
 
 .image-down {
   position: absolute;
-  width: 320px;
-  height: 160px;
+  width: 375px;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
   left: 0;
   top: 540px;
 }
+
+.projects {
+  display: grid;
+}
+
+.component {
+  /* Rectangle 21 */
+
+  position: absolute;
+  width: 375px;
+  left: 0;
+  margin-top: -274px;
+  border: 0.3333px solid rgb(255 255 255 / 10%);
+}
+
+.box {
+  /* Rectangle 22 */
+  position: absolute;
+  width: 375px;
+  height: 190px;
+  left: -1px;
+  background: #1a2236;
+  border: 0 solid rgb(255 255 255 / 10%);
+  backdrop-filter: blur(0.5px);
+
+  /* Note: backdrop-filter has minimal browser support */
+}
+
+.one {
+  top: 1048px;
+}
+
+.two {
+  top: 1456px;
+}
+
+.three {
+  top: 1862px;
+}
+
+.four {
+  top: 2268px;
+}
+
+.five {
+  top: 2674px;
+}
+
+.six {
+  top: 3080px;
+}
+
+h3 {
+  /* Project name goes here */
+
+  width: 264px;
+  height: 16px;
+  left: 40px;
+  font-family: Poppins, Georgia, "Times New Roman", Times, serif;
+  font-style: normal;
+  font-weight: 100;
+  font-size: 20px;
+  line-height: 80%;
+  padding-left: 20px;
+
+  /* identical to box height, or 16px */
+
+  color: #fff;
+}
+
+.keywords li {
+  list-style: none;
+  display: inline-flex;
+  background-color: #98dede;
+  height: 20px;
+  left: 20px;
+  font-family: Poppins, Georgia, "Times New Roman", Times, serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.333px;
+  padding-left: 10px;
+  padding-top: 8px;
+  padding-right: 10px;
+  margin-left: -18px;
+  margin-right: 20px;
+}
+
+.buttonbox {
+  width: 240px;
+  height: 50px;
+  background: #1a2236;
+  border: 0.5px solid rgb(255 255 255 / 10%);
+  backdrop-filter: blur(0.5px);
+  margin-left: 20px;
+  font-family: Poppins, Georgia, "Times New Roman", Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  text-align: center;
+  line-height: 24px;
+
+  /* identical to box height */
+
+  align-content: left;
+  color: #fff;
+}
+
+.buttonbox img {
+  align-items: center;
+}
+
+.about h1 {
+  display: flex;
```

Comment:
- Good job for using `flex`💪 , I think you used it in the un appropriate place for specific `h1`, please use it to arrange technologies categories as your previous reviewer suggested.👍👌